app/javascript/admin/controllers/config_controller.js

```diff
@@ -93,6 +93,39 @@ export default class ConfigController extends Controller {
     }
   }
 
+  displaySnackbar(message) {
+    return document.dispatchEvent(
+      new CustomEvent('snackbar:add', {
+        detail: { message },
+      }),
+    );
+  }
+
+  async activateLightConfirmation(event) {
+    event.preventDefault();
+    try {
+      const body = new FormData(event.target);
+      const response = await fetch(event.target.action, {
```

Comment:
I had proposed using our `request utility` class, but it requires a refactor to be able to use it in the admin area, so for this PR, let's keep it as `fetch` for now.